istioctl/cmd/testdata/authz/converter/error-no-clusterRbacConfig.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: "rbac.istio.io/v1alpha1"
```

Comment:
Question: If you use "-f" option to specify input yamls, do you have to include "ClusterRbacConfig"?

Reply:
By default we require the ClusterRbacConfig, user could use `--allowNoClusterRbacConfig` to make it optional.